[2025-02-18T11:29:32.623+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T11:29:32.649+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dynamic_task_mapping_employee_experience_v3.validate_employee scheduled__2025-02-17T00:00:00+00:00 map_index=3 [queued]>
[2025-02-18T11:29:32.662+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dynamic_task_mapping_employee_experience_v3.validate_employee scheduled__2025-02-17T00:00:00+00:00 map_index=3 [queued]>
[2025-02-18T11:29:32.662+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-02-18T11:29:32.680+0000] {taskinstance.py:2889} INFO - Executing <Mapped(_PythonDecoratedOperator): validate_employee> on 2025-02-17 00:00:00+00:00
[2025-02-18T11:29:32.688+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1357) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T11:29:32.688+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dynamic_task_mapping_employee_experience_v3', 'validate_employee', 'scheduled__2025-02-17T00:00:00+00:00', '--job-id', '549', '--raw', '--subdir', 'DAGS_FOLDER/taskmapp.py', '--cfg-path', '/tmp/tmpl9zbwzv9', '--map-index', '3']
[2025-02-18T11:29:32.690+0000] {standard_task_runner.py:105} INFO - Job 549: Subtask validate_employee
[2025-02-18T11:29:32.690+0000] {standard_task_runner.py:72} INFO - Started process 1367 to run task
[2025-02-18T11:29:32.741+0000] {task_command.py:467} INFO - Running <TaskInstance: dynamic_task_mapping_employee_experience_v3.validate_employee scheduled__2025-02-17T00:00:00+00:00 map_index=3 [running]> on host 063e621c401a
[2025-02-18T11:29:32.842+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='rithvik' AIRFLOW_CTX_DAG_ID='dynamic_task_mapping_employee_experience_v3' AIRFLOW_CTX_TASK_ID='validate_employee' AIRFLOW_CTX_EXECUTION_DATE='2025-02-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-17T00:00:00+00:00'
[2025-02-18T11:29:32.843+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-18T11:29:32.846+0000] {python.py:240} INFO - Done. Returned value was: {'id': 4, 'name': 'Aaryan Patel', 'experience': 10, 'department': 'Marketing'}
[2025-02-18T11:29:32.866+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-18T11:29:32.867+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dynamic_task_mapping_employee_experience_v3, task_id=validate_employee, run_id=scheduled__2025-02-17T00:00:00+00:00, map_index=3, execution_date=20250217T000000, start_date=20250218T112932, end_date=20250218T112932
[2025-02-18T11:29:32.904+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T11:29:32.913+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
